tests/project/test_data.py

```diff
@@ -13,6 +13,7 @@
 class TestProjectData(PyironTestCase):
     @classmethod
     def setUpClass(cls):
+        super().setUpClass()
```

Comment:
I am happy to include this here, however, this would is not required for the functionality, is it?

Reply:
Yeah, that was just because I was at it already anyway.